collector/crio/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM warmmetal/kindest-base-crio:latest
```

Comment:
Seems like you could just mount this file with a docker-compose file so it's just a pull without building a new layer. Did you try that?